ztp/install-accelerator/manifests/clusterrole.yaml

```diff
@@ -0,0 +1,44 @@
+kind: ClusterRole
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name: ztp-profile-accelerator-clusterrole
+rules:
+# Read cluster version, clusteroperators, configmaps
+- apiGroups:
+  - config.openshift.io
+  - ""
+  resources:
+  - clusterversions
+  - clusteroperators
+  - configmaps
+  verbs:
+  - get
+  - list
+# Approve installplans
+- apiGroups:
+  - operators.coreos.com
+  resources:
+  - installplans
+  verbs:
+  - get
+  - list
+  - watch
+  - patch
+# Do anything with custom resources we manage
+- apiGroups:
```

Comment:
Yep, that's correct. I will need to add all the ones we support for DU, including FEC. Thanks for pointing that out!

Reply:
Np. Just curious, in theory, is it possible for the manifest extractor to take/infer this kind of info from the policies and use it here?

Reply:
Yeah, good idea! The only problem I see is mapping GVK to GVR. For REST mapper to work on the hub we need all these CRDs to be installed there. Another option is to create a hard-coded mapping between Kind and Resource for all the custom resources we support, look it up and add to the ClustreRole, but this is again hard-coding, just in a different place, but it's more secure because it will enable only the required resources for this particular installation.

Reply:
The config policy controller actually uses a role that has access to *. The point of this service is to do the same thing before the policy stuff is ready. We could argue it's entitled to the same kind of access? 

```
kind: ClusterRole
apiVersion: rbac.authorization.k8s.io/v1
metadata:
  name: 'open-cluster-management:config-policy-controller'
  uid: ce1aca3d-81bf-4472-b79f-7d4c01594030
  resourceVersion: '16750'
  creationTimestamp: '2023-09-11T19:15:28Z'
  labels:
    addon.open-cluster-management.io/hosted-manifest-location: hosting
    app: config-policy-controller
    chart: config-policy-controller-2.2.0
    heritage: Helm
    release: config-policy-controller
rules:
  - verbs:
      - '*'
    apiGroups:
      - '*'
    resources:
      - '*'
  - verbs:
      - '*'
    nonResourceURLs:
      - '*'
```

Reply:
Done :)